WIP-CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - predictlinear alerts
 - Calico-accountant is now being scheduled on master nodes.
 - it is now possible to set tolerations and affinity for vulnerability-exporter
+- removed continue_if_exception that was set to False for curator job to not fail when a snapshot is in progress and it is trying to remove some indices.
```

Comment:
No need for a changelog entry for this fix
```suggestion
```

Reply:
Removed it!!